TreeMaker/python/maker.py

```diff
@@ -56,12 +59,26 @@ def __init__(self,parameters):
         self.getParamDefault("applybaseline", False)
         self.getParamDefault("saveMinimalGenParticles", True)
 
+        # branch control options
+        self.getParamDefault("includeBranches", "")
+        self.getParamDefault("excludeBranches", "")
+        self.getParamDefault("exactBranches", False)
+        if len(self.includeBranches)>0 and len(self.excludeBranches)>0:
+            raise ValueError("includeBranches and excludeBranches are exclusive options; pick one!")
+        for branchListName in ["includeBranches","excludeBranches"]:
+            branches = getattr(self,branchListName)
+            if not isinstance(branches,list) and branches.endswith(".txt"):

```

Comment:
Why not make the list a python list in a file which can be imported? Parsing a text file just seems like an added complication, not that it's super hard (obviously).

Reply:
In this case, I don't think importing an arbitrary python file is easier than importing a text file (which is intended to have exactly one branch name per line).